generators/angular/generator.ts

```diff
@@ -213,9 +244,9 @@ export default class AngularGenerator extends BaseApplicationGenerator {
 
   get default() {
     return this.asDefaultTaskGroup({
-      loadEntities() {
+      loadEntities({ application }) {
         const entities = this.sharedData.getEntities().map(({ entity }) => entity);
-        this.localEntities = entities.filter(entity => !entity.builtIn && !entity.skipClient);
+        application.frontendEntities = entities.filter(entity => !entity.builtIn && !entity.skipClient);
```

Comment:
`localEntities` was an attempt to don’t use needles. In v8, needles are more stable and are been used heavily.

We should not expose it at application. Frontend entities are more complex than this filter.

`localEntites` should be dropped in favor of needles implementation.

Reply:
I kept it like this (just modifying the attribute name): will be part of another PR (or as it is, which I find more solid than needle)